README.md

```diff
@@ -3,20 +3,70 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-{Project tag line}
+Chinook.Navigation
 
-{Small description of the purpose of the project}
+This library offers a simple method of navigating through an app. [StackNavigation](src/StackNavigation.Abstractions/StackNavigation.md) and [SectionsNavigation](src/SectionsNavigation.Abstractions/SectionsNavigation.md) are used together to provide a simple interface to navigate forward, backward and between discrete sections.
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
 ## Getting Started
 
-{Instructions to quickly get started using the project: pre-requisites, packages
-to install, sample code, etc.}
+### Uno project
+
+To get started in a project which uses the [Uno platform](https://platform.uno/):
+
+* Add to your project the `Chinook.SectionsNavigation.Uno` nuget package and its dependencies.
+* Create registrations of type `Dictionary<Type, Type>`, which `FrameSectionsNavigator` will use for navigation. These registrations represent the link between the ViewModel and the View to display in the navigation. For instance, if you want to display a FavouritesPage and bind it to a FavouritesPageViewModel, make sure your registrations dictionary contains a KeyValuePair of <typeof(FavouritesPageViewModel), typeof(FavouritesPage)>. Repeat for every page to which your app can navigate. For more information on how ViewModels are used in the navigation, please look at the documentation for [StackNavigation](src/StackNavigation.Abstractions/StackNavigation.md).
+* With these registrations, create a singleton instance of `IStackNavigator` which will be used throughout the app:
```

Comment:
If you install `Chinook.SectionsNavigation`, you'll probably use a `SectionsNavigator` and not a `IStackNavigator`.